sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -204,7 +207,7 @@ foreach (RecognizedForm form in forms)
 ```
 
 ### Recognize Receipts
-Recognize data from US sales receipts using a prebuilt model.
+Recognize data from US sales receipts using a prebuilt model. Receipt fields recognized by the service can be found [here][service_recognize_receipt_fields].
```

Comment:
```suggestion
Recognize data from sales receipts using a prebuilt model. Receipt fields recognized by the service can be found [here][service_recognize_receipt_fields].
```

Reply:
I made the modifications in https://github.com/Azure/azure-sdk-for-net/pull/16026
Thanks!